### PR TITLE
more links to shiny examples

### PR DESCRIPTION
**Pull request by stla**
I added some links in `shiny.Rmd` (at the end). I knitted to `html`, I don't know if I had to do that. This has changed the file `shiny.html` which I commited too, and there are many differences as compared to the original file.

### CODE REVIEW

File: shiny.Rmd [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Using DT in Shiny
+title: "Using DT in Shiny"
```
Comment (yihui) [inline shiny.Rmd]: An unnecessary [cosmetic change](https://yihui.name/en/2018/11/cosmetic-changes/) but that's fine this time.

File: shiny.Rmd [hunk 2]
```diff
@@ -145,3 +145,7 @@ You can edit a table via the `editable` argument of `datatable()`. After you fin
 Below is a list of miscellaneous examples:
 
 1. [Radio Buttons in Tables](011-radio.html)
+
+2. [Useful callbacks for DT (in Shiny)](https://laustep.github.io/stlahblog/posts/DTcallbacks.html)
+
+3. [Plotting the columns of a Datatable](https://laustep.github.io/stlahblog/posts/PlotDatatableColumns.html)
```
Comment (yihui) [inline shiny.Rmd]: Please use `1. ` for all items to minimize future GIT changes (e.g. when we need to insert a new item before a certain existing item).

File: shiny.html [hunk 3]
```diff
@@ -7,64 +7,149 @@
 <meta charset="utf-8" />
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <meta name="generator" content="pandoc" />
-<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />
```
Comment (yihui) [inline shiny.html]: Please discard all changes in the generated output. We'll compile the Rmd by ourselves.

File: shiny.Rmd (hunk 1 above)
Comment (stla) [reply shiny.Rmd]: Strange, I didn't change that.

File: shiny.html (hunk 3 above)
Comment (stla) [reply shiny.html]: So it would be easier for me if you rejected this pull request and I do a new one.

Comment (stla) [reply shiny.html]: Done. I have deleted the fork and I have reforked. Please reject this pull request, I'm going to do the new one.

**Pull request closed without merging**